### PR TITLE
Add /certified page

### PR DESCRIPTION
**Pull request by britneywwc**
## Done

- Added new /certified page
- Figma: https://www.figma.com/design/tI8YK4wQnKiWLAxdX1fWnB/24.10-jp.ubuntu.com?node-id=0-1&t=Ey9MQHs6ERH0AKQV-0
- Docs: https://docs.google.com/document/d/1aNshSU4yZ8otlPoWTjHmO8lq1gY92J1T4w5iVK3pzdI/edit

## QA

- Go to [demos](https://jp-ubuntu-com-482.demos.haus/certified)
- Check that the /certified webpage adheres to design and docs

## Issue / Card

Fixed [WD-10813](https://warthogs.atlassian.net/browse/WD-10813)

## Screenshots

[if relevant, include a screenshot]

[WD-10813]: https://warthogs.atlassian.net/browse/WD-10813?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by akbarkz** [comment]
@britneywwc Since there is this new webpage for /certified, shouldn't you point links on the landing page's header and footer to it instead of to ubuntu.com/certified one?

**Review by akbarkz — APPROVED** [review]

**Comment by juanruitina** [comment]
- [ ] Please add the copy doc metatag https://docs.google.com/document/d/1aNshSU4yZ8otlPoWTjHmO8lq1gY92J1T4w5iVK3pzdI/edit
- [ ] The Ubuntu Certified "sticker" looks weird right-aligned on mobile. @lyubomir-popov what do you think we should do about it? Left, center align it? I wouldn't hide it, it's meaningful.

@britneywwc Please run it by Yvonne to make sure we didn't mess with the content. Lyubo should also have a look at this, given it's a new page (I assigned the PR to him).

**Comment by lyubomir-popov** [comment]
the footer needs `is-dark` otherwise the background is wrong and the horizontal rules are not visible 
image
couple of logos need fixing as well

File: templates/certified.html [hunk 1]
```diff
@@ -0,0 +1,595 @@
...
+
+{% block title %}認証済みH/W{% endblock %}
+{% block meta_description %}認証済みのデバイスは信頼性とパフォーマンスのテストに合格しているため、Ubuntuが即座に力を発揮します。{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1aNshSU4yZ8otlPoWTjHmO8lq1gY92J1T4w5iVK3pzdI/edit{% endblock meta_copydoc %}
+{% block body_class %}is-paper{% endblock %}
+
+{% block outer_content %}
+<section class="p-section--hero">
+  <div class="row">
+    <div class="col-9">
```
Comment (lyubomir-popov) [inline templates/certified.html:12]:
```suggestion
    <div class="col-9 col-medium-5">
```

File: templates/certified.html [hunk 2]
```diff
@@ -0,0 +1,595 @@
...
+      <div class="p-section--shallow">
+        <h1>Ubuntu認証プログラム</h1>
+        <h2>統合テストに合格した認証済みのハードウェアでUbuntuを最大限に活用</h2>
+      </div>
+      <div class="p-section--shallow">
+        <hr />
+        <p><a href="/contact-us" class="p-button--positive">お問い合わせ</a></p>
+      </div>
+    </div>
+    <div class="col-3 u-align--right">
```
Comment (lyubomir-popov) [inline templates/certified.html:22]:
```suggestion
    <div class="col-3col-medium-1 u-hide--small u-align--right">
```

**Review by lyubomir-popov — COMMENTED** [review]
couple of suggestions for the hero on medium screens

**Pull request merged**